### DISCUSSION TITLE
Does BotSORT run with gmc activated?

### DISCUSSION THREAD

**Discussion by ulrikabremberg** [mikel-brostrom/yolo_tracking#774] [Q&A] [answered]
Hi! Great repository :star:
How can I know if BotSORT runs with gmc activated? Saw in one of the experiments that it had been deactivated due to speed there, and now I'm curious if it was only for that experiment.

**Comment by mikel-brostrom** [comment, marked as answer]
Hej!

You can check

https://github.com/mikel-brostrom/yolov8_tracking/blob/03fc3dfcfe33f8a363b5b9a4b776e38060f8a367/trackers/botsort/bot_sort.py#L242

This will activate different camera motion compensations

**Reply by ulrikabremberg** [reply]
Okay, so if I have that line in my botsort file I can assume it's activated then? :) Thanks for the quick answer!

**Reply by mikel-brostrom** [reply]
Yup :smile:

**Reply by mikel-brostrom** [reply]
`gmc` is always used by default

https://github.com/mikel-brostrom/yolov8_tracking/blob/03fc3dfcfe33f8a363b5b9a4b776e38060f8a367/trackers/botsort/bot_sort.py#L332
